scripts/save_org_product_data_daily.pl

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/perl -w
+
+# This file is part of Product Opener.
+#
+# Product Opener
+# Copyright (C) 2011-2023 Association Open Food Facts
+# Contact: contact@openfoodfacts.org
+# Address: 21 rue des Iles, 94100 Saint-Maur des Fossés, France
+#
+# Product Opener is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+use ProductOpener::PerlStandards;
+
+use CGI::Carp qw(fatalsToBrowser);
+
+use ProductOpener::Config qw/:all/;
+use ProductOpener::Data qw/:all/;
+use ProductOpener::Paths qw/%BASE_DIRS ensure_dir_created/;
+use ProductOpener::Users qw/$Owner_id/;
+use ProductOpener::Orgs qw/retrieve_org/;
+use Storable qw(store);
+
+# This script is run daily to gather organisation data 
+# such as number of products, number of products with errors etc,
+
+ensure_dir_created($BASE_DIRS{ORGS});
+
+my $products_collection = get_products_collection();
+my $orgs_collection = get_orgs_collection();
+
+sub update_org_data {
+    my $org_id = shift;
```

Comment:
Please follow our standards and use named parameters.

Reply:
I'm not sure what you mean @alexgarel

Reply:
Never mind, I get it now.